lua/autorun/server/sv_autoclean.lua

```diff
@@ -20,11 +15,8 @@ local function notifyPlayers( notification )
 end
 
 local function runCleanupCommandsOnPlayers()
-    for command, _ in pairs( clientCleanupCommands ) do
-        net.Start( "CFC_AutoClean_RunCommand" )
-        net.WriteString( command )
-        net.Broadcast()
-    end
+    net.Start( "CFC_RunAutoClean" )
```

Comment:
you need to update  `util.AddNetworkString( "CFC_AutoClean_RunCommand" )
other than that it looks good 👍